### PR TITLE
Add release dates to Changelog.

### LINKED ISSUE #875: Add release dates to changelog

### PR DESCRIPTION
**Pull request by pranaysashank**
Closes #875

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -393,14 +393,14 @@ concurrently using this.
...
-## 0.2.1
+## 0.2.1 - 5th June 2018
 
 ### Bug Fixes
 * Fixed a bug that caused some transformation ops to return incorrect results
   when used with concurrent streams. The affected ops are `take`, `filter`,
   `takeWhile`, `drop`, `dropWhile`, and `reverse`.
 
-## 0.2.0
+## 0.2.0 - 13th May 2018
```
Comment (adithyaov) [inline Changelog.md:403]: 14th May according to https://github.com/composewell/streamly/releases

File: Changelog.md [hunk 2]
```diff
@@ -365,7 +365,7 @@ concurrently using this.
 * Add `takeWhileM` and `dropWhileM`
 * Add `filterM`
 
-## 0.3.0
+## 0.3.0 - 19th June 2018
```
Comment (adithyaov) [inline Changelog.md:368]: 15th June according to https://github.com/composewell/streamly/releases

File: Changelog.md [hunk 3]
```diff
@@ -286,7 +286,7 @@ concurrently using this.
 * Performance improvements
 * Add benchmarks to measure composed and iterated operations
 
-## 0.5.2
+## 0.5.2 - 26th October 2018
```
Comment (adithyaov) [inline Changelog.md:289]: 27th? https://github.com/composewell/streamly/releases

File: Changelog.md (hunk 2 above)
Comment (pranaysashank) [reply Changelog.md:368]: Hackage says 19th June, (maybe docs were uploaded manually and it uses that as the uploaded date?)

File: Changelog.md (hunk 1 above)
Comment (pranaysashank) [reply Changelog.md:403]: I used Hackage upload dates which are likely UTC so a few hours behind

**Comment by adithyaov** [comment]
You can decide on which of the dates to include. Minor inconsistencies are ok, can be ignored.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]
1. Perhaps use just months instead of exact dates
2. Perhaps use parenthesis ` ## 0.7.2 (April 2020)`

**Pull request merged**